src/Utilities/TextDiff.jl

```diff
@@ -87,15 +87,11 @@ end
 prefix(::Diff{Lines}, s::Symbol) = s === :green ? "+ " : s === :red  ? "- " : "  "
 prefix(::Diff{Words}, ::Symbol) = ""
 
-function showdiff(io::IO, diff::Diff)
+function Base.show(io::IO, diff::Diff)
+    get(io, :color, false) || println(io, "Warning: Diff output requires color.")
```

Comment:
Any easy way we could maybe fall back to the line-by-line output if there is no color?

Reply:
Hmm. I included `Output:` and `Expected output:` so witht those I think it is fine not to show the diff again.

Reply:
Do you mean completely removing the outputting of the `diff` object? Or just when there is no color?

Reply:
No I just mean that in case of no color I don't think it helps much printing line by line when I already added both the output and expected output.